### PR TITLE
Add design patterns, content guidance and information security guidelines

### PR DESCRIPTION
**Pull request by henryhadlow**

### CODE REVIEW

File: app/templates/views/design-patterns-content-guidance.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+    <section id="content-guidance">
+      <h2 class="heading-medium">Content guidance</h2>
+
+      <h3 class="heading-small" id="include-one-important-message">Include one important message</h3>
+
+      <p>Don&#8217;t explain the whole process – just tell the user what will happen next.</p>
+
+      <ul class="list list-bullet">
+        <li>If the user needs to do something, tell them what they have to do. Include a link if possible – see our <a href="">Information security guidelines</a></li>
```
Comment (quis) [inline app/templates/views/design-patterns-content-guidance.html]: Missing `href`

File: app/templates/views/information-security.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        <li><a href="#you-can-do-more">You can do more if you want to</a></li>
+      </ul>
+    </section>
+
+    <section id="start-with-needs">
+      <h2 class="heading-medium">Start with needs – user needs, not government needs</h2>
+
+      <p>Start by writing the message you want to send. Don’t worry about the information security aspect just yet – write the message you want to convey as clearly and directly as possible.</p>
+
+      <p>We have <a href="">design patterns</a> and <a href="">content guidance</a> to help you write clearly and convey the right information at the right time.</p>
```
Comment (quis) [inline app/templates/views/information-security.html]: Ditto

File: app/templates/views/information-security.html [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+      <h3 class="heading-small" id="guideline-phishing">Don’t send requests for personal information of any kind, unless the request is directly connected with a transaction</h3>
+
+      <p>To reduce the risk from phishing attacks, don’t send <strong>requests</strong> for personal information <strong>of any kind</strong>, unless the request is <strong>directly connected with a transaction</strong>.</p>
+
+      <p>It’s OK to send a request for personal information if it’s directly connected with a transaction. Here are two examples of where it would be OK:</p>
+
+      <ul class="list list-bullet">
+        <li>Someone clicks a ‘Forgot your password?’ link – it’s OK to send them a link where they can reset their password</li>
+        <li>Someone’s MOT is about to expire – y</li>
```
Comment (quis) [inline app/templates/views/information-security.html]: Unfinished sentence?

**Comment by yahoopete** [comment]
Closing this - needs some work.  Will revisit shortly.

**Pull request closed without merging**